catalog/lib/theme/app_buttons.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:catalog/theme/app_text_styles.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:catalog/theme/app_colors.dart';
+
+typedef AppFilledButton = FilledButton;
+typedef AppStrokeButton = OutlinedButton;
+typedef AppGhostButton = TextButton;
+
+class AppButtonsStyle {
+  final ButtonStyle filledButton = _appFilledButton;
+  final ButtonStyle outlineButton = _appOutlineButton;
+  final ButtonStyle textButton = _appTextButton;
+  final ButtonStyle secondaryFilledButton = _appSecondaryFilledButton;
+  final ButtonStyle secondaryOutlineButton = _appSecondaryOutlineButton;
+  final ButtonStyle secondaryTextButton = _appSecondaryTextButton;
+
+  AppButtonsStyle();
+
+  static AppButtonsStyle getButtonTheme() => AppButtonsStyle();
+}
+
+final _appFilledButton = FilledButton.styleFrom(
+  minimumSize: Size(double.infinity, 50.h),
+  shape: RoundedRectangleBorder(
+    borderRadius: BorderRadius.circular(16.r),
+    side: const BorderSide(
+      width: 2,
+      color: Colors.transparent,
+    ),
+  ),
+  textStyle: AppTextStyles.getAppStyles().buttonLarge,
+  elevation: 0.0,
+  foregroundColor: AppColors.getColorScheme().textColor.shade100,
+);
+
+final _appOutlineButton = OutlinedButton.styleFrom(
+  minimumSize: Size(double.infinity, 50.h),
+  shape: RoundedRectangleBorder(
+    borderRadius: BorderRadius.circular(16.r),
+  ),
+  side: BorderSide(
+    width: 2,
+    color: AppColors.getColorScheme().primary,
+  ),
+  textStyle: AppTextStyles.getAppStyles().buttonLarge,
+  elevation: 0.0,
+);
+
+final _appTextButton = TextButton.styleFrom(
+  minimumSize: Size(double.infinity, 50.h),
+  shape: RoundedRectangleBorder(
```

Comment:
There are no borders since it's all transparent by default and uses stadium border by default, so the same goes for the border radius.
This is the default style implementation as example:
https://api.flutter.dev/flutter/material/TextButton/defaultStyleOf.html

```
@override
ButtonStyle defaultStyleOf(BuildContext context) {
  final ThemeData theme = Theme.of(context);
  final ColorScheme colorScheme = theme.colorScheme;

  return Theme.of(context).useMaterial3
    ? _TextButtonDefaultsM3(context)
    : styleFrom(
        foregroundColor: colorScheme.primary,
        disabledForegroundColor: colorScheme.onSurface.withOpacity(0.38),
        backgroundColor: Colors.transparent,
        disabledBackgroundColor: Colors.transparent,
        shadowColor: theme.shadowColor,
        elevation: 0,
        textStyle: theme.textTheme.labelLarge,
        padding: _scaledPadding(context),
        minimumSize: const Size(64, 36),
        maximumSize: Size.infinite,
        shape: const RoundedRectangleBorder(borderRadius: BorderRadius.all(Radius.circular(4))),
        enabledMouseCursor: SystemMouseCursors.click,
        disabledMouseCursor: SystemMouseCursors.basic,
        visualDensity: theme.visualDensity,
        tapTargetSize: theme.materialTapTargetSize,
        animationDuration: kThemeChangeDuration,
        enableFeedback: true,
        alignment: Alignment.center,
        splashFactory: InkRipple.splashFactory,
      );
}
```